styles/navigations-all.css

```diff
@@ -0,0 +1,125 @@
+.component-nav1 {
+  width: 95%;
+  height: 100%;
+  justify-self: center;
+}
+
+.component-nav1 > nav {
+  border-radius: 1rem;
+  color: white;
+  padding: 0.5rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #011627;
+}
+
+.component-nav1 > nav > img {
+  width: 40px;
+  height: 40px;
+}
+
+.nav1-list > ol {
+  display: flex;
+  height: 100%;
+}
+
+.nav1-list > ol > li {
+  margin: 0rem 1rem;
+  list-style: none;
+  cursor: pointer;
+}
+
+.nav1-user {
+  display: flex;
+  height: 100%;
+  width: 23%;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.nav1-user-account {
+  padding: 0.3rem 0.5rem;
+  background: #011627;
+  color: white;
+  border: none;
+  font-size: 1rem;
+  cursor: pointer;
+}
+
+.signout-btn {
+  background: #fdd405;
+  border-radius: 10px;
+  padding: 0.5rem 1rem;
+  color: black;
+}
+
+.nav1-user > i {
+  color: #fdd405;
+}
+
+/* CSS FOR NAV 2 */
+
+.component-nav2 {
+  width: 95%;
+  height: 100%;
+  justify-self: center;
+}
+
+.component-nav2 > nav {
+  padding: 0.5rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: white;
+}
+
+.component-nav2 > nav > img {
+  width: 100px;
+  height: 50px;
+}
+
+.nav2-list > ol {
+  display: flex;
+  height: 100%;
+}
+
+.nav2-list > ol > li {
+  margin: 0rem 1rem;
+  list-style: none;
+  font-size: 1rem;
+  font-weight: 600;
+  color: #74747f;
+  cursor: pointer;
+}
```

Comment:
feat: when adding animations can add transition property with some delays to make animations feel smooth